pkg/controller/node/nodecontroller.go

```diff
@@ -740,14 +740,13 @@ func (nc *NodeController) handleDisruption(zoneToNodeConditions map[string][]*v1
 		// We're switching to full disruption mode
 		if allAreFullyDisrupted {
 			glog.V(0).Info("NodeController detected that all Nodes are not-Ready. Entering master disruption mode.")
-			for i := range nodes {
+			for _, node := range nodes {
```

Comment:
Yup. We generally prefer using `for i := ...`, as this doesn't involve copying object unnecessarily.